pyproject.toml

```diff
@@ -305,3 +307,16 @@ notes = """,
 [tool.pylint.typecheck]
 # @patch changes the signature of a function it's applied to; don't raise "no-value-for-parameter" here
 signature-mutators=["unittest.mock.patch"]
+
+[tool.towncrier]
+package = "nautobot"
+directory = "changes"
+filename = "nautobot/docs/release-notes/version-1.4.md"
+template = "towncrier_template.j2"
+start_string = "<!-- towncrier release notes start -->"
+issue_format = "[#{issue}](https://github.com/nautobot/nautobot/issues/{issue})"
```

Comment:
So this _should_ be OK for issue-less PRs (since GitHub does a seamless redirect to the appropriate subdirectory) but is there a way to do something like PR#1234 to write the proper links in those situations?

Reply:
This is [hardcoded to use python `.format()`](https://github.com/twisted/towncrier/blob/6c94dc2c90febe1c6353041462e70833028ad15f/src/towncrier/_builder.py#L222) in towncrier so we wouldn't be able to have multiple formats unless we use a different solution.